python/interpret_community/mimic/model_distill.py

```diff
@@ -76,7 +77,10 @@ def _model_distill(teacher_model_predict_fn, uninitialized_surrogate_model, data
         surrogate_model = uninitialized_surrogate_model(**explainable_model_args)
     if is_classifier and teacher_y.shape[1] == 2:
         # Make sure output has only 1 dimension
-        teacher_y = teacher_y[:, 1]
+        if isinstance(teacher_y, pd.DataFrame):
```

Comment:
the teacher_y can be a DataFrame?  In that case, I would recommend to just change this code:
```
teacher_y = teacher_model_predict_fn(original_training_data)
```
to convert teacher_y to numpy right after that line.
Otherwise, the multiclass surrogate code will probably break as well.

Reply:
fixed